released/packages/coq-commutative-diagrams/coq-commutative-diagrams.0.1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+synopsis: "A Coq plugin to deal with commutative diagrams"
+maintainer: ["luc@dwarfmaster.net"]
+authors: ["Luc Chabassier"]
+license: "MIT"
+homepage: "https://github.com/dwarfmaster/commutative-diagrams"
+bug-reports: "https://github.com/dwarfmaster/commutative-diagrams/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {= "4.12.1"}
+  "coq" {= "8.15.2"}
```

Comment:
Are these dependencies that strict?

Reply:
We typically put a lower and upper bound for Coq, and, if needed, a lower bound on OCaml (knowing that 8.15 requires 4.05 IIRC).

Reply:
Looking at the code you want 4.08 at least (for monadic let)

Reply:
For `coq` I think so, or at least it needs `= "8.15"`, since I'm pretty sure some things changed in the ocaml API since `8.16`. For ocaml indeed it needs to be at least `4.08` for the let, and it needs to be the exact version coq is compiled with, but I guess opam guarantees that anyway ?

Reply:
That's fine by me. Since I had quite a lot of bugs linked to mismatched versions of coq, I may have been too conservative in the versions constraints.

Reply:
Yes, we refrain from changing the ocaml APIs in minor release, so all 8.15.x have the same APIs.

>  I guess opam guarantees that anyway

Correct